src/controllers/userController.js

```diff
@@ -11,6 +13,30 @@ const signup = catchAsync(async (req, res) => {
   res.status(201).json({ message: "SIGNUP_SUCCESS" });
 });
 
+
+const signin = catchAsync(async (req, res) => {
+	const { email, password } = req.body;
+	if (!email || !password) detectError("NO INPUT", 404);
+
+	const accessToken = await userService.signIn(email, password);
+
+	res.status(201).json({ accessToken: accessToken });
+});
+
+
+const oAuth = catchAsync(async (req, res) => {
```

Comment:
```suggestion
const googleLogin = catchAsync(async (req, res) => {
```
oauth를 이용한 로그인은 얼마든지 추가될 수 있으니 고려하여 네이밍 해주세요!